docs/CONFIG-VARS.md

```diff
@@ -428,3 +429,14 @@ postgres_servers = {
   }
 }
 ```
+
+## Cluster Logging
+
+[CloudWatch](https://aws.amazon.com/cloudwatch/) can be enabled by setting the `cluster_enabled_log_types` configuration variable. The list of audits will be streamed to the CloudWatch Log Group.
+
+**NOTE**: This requires additional IAM policies to create and tag CloudWatch logs.
+
+<!--| Name | Description | Type | Default | Notes | -->
+| <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
+| :--- | :--- | :--- | :--- | :--- |
+| cluster_enabled_log_types | List of audits to record from EKS cluster in CloudWatch | list(string) | |  |
```

Comment:
If you can find a link to the AWS doc that lists supported values for the string list it would be helpful to add it to the Notes column of this table.